bridge/client/app/app-header/app-header.component.html

```diff
@@ -59,19 +59,19 @@ <h3 class="m-0">Get started</h3>
         <p class="m-0">Version check</p>
       </div>
       <div>
-        <dt-switch (change)="versionCheckClicked($event)" [checked]="keptnInfo?.versionCheckEnabled"></dt-switch>
+        <dt-switch (change)="versionCheckClicked($event)" [checked]="keptnInfo?.versionCheckEnabled !== false"></dt-switch>
       </div>
     </div>
-    <p class="m-0 mt-2 small">By enabling this feature, Keptn will collect statistical data and notify about new version and security patches for Keptn and Keptn Bridge. Details can be found at <a [href]="versionCheckReference | keptnUrl" target="_blank" [textContent]="versionCheckReference | keptnUrl"></a></p>
+    <p class="m-0 mt-2 small">By enabling this feature, Keptn will notify about new versions and security patches for Keptn and therefore collect statistical data. Details can be found at <a [href]="versionCheckReference | keptnUrl" target="_blank" [textContent]="versionCheckReference | keptnUrl"></a></p>
```

Comment:
can we add "recommended" to this?

"By enabling this feature (recommended), Keptn will notify about new versions and security patches for Keptn and therefore collect statistical data."